src/Base/Quantity.pyi

```diff
@@ -0,0 +1,107 @@
+from Metadata import export, constmethod
+from PyObjectBase import PyObjectBase
+from typing import overload, Final, Tuple, Union
+from Unit import Unit as UnitPy
+
+@export(
+    NumberProtocol=True,
+    RichCompare=True,
+    Constructor=True,
+    Delete=True,
+)
+class Quantity(PyObjectBase):
+    """
+    Quantity
+    defined by a value and a unit.
+
+    The following constructors are supported:
+    Quantity() -- empty constructor
+    Quantity(Value) -- empty constructor
+    Quantity(Value,Unit) -- empty constructor
```

Comment:
Are you fixing things like this as you come across them? Obviously those aren't empty constructors.